src/main/flight/pid.c

```diff
@@ -670,6 +686,9 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     dtermCutLowpassHz = pidProfile->dterm_cut_lowpass_hz;
     dtermCutGain = pidProfile->dterm_cut_gain * dtermCutPercent * D_CUT_GAIN_FACTOR / dtermCutLowpassHz;
     // lowpass included inversely in gain since stronger lowpass decreases peak effect
+#endif
+#if defined(USE_AIRMODE_LPF)
```

Comment:
If this is ifdef block is moved below the `dtermCutFactor = 1.0f` it will rebase more cleanly.

Reply:
@fujin is this still an issue? Looks like it's holding up merging.